### DISCUSSION TITLE
React native: how to insert image?

### DISCUSSION THREAD

**Discussion by JulianHSC** [microsoft/react-native-windows#12921] [Q&A] [answered]
Hello team,
I followed the code in https://www.tutorialspoint.com/react_native/react_native_images.htm and got error: Error: Unable to resolve module. How do i insert images for react native? Can someone point?

**Comment by chrisglein** [comment, marked as answer]
I recommend looking at the official RN docs: https://reactnative.dev/docs/image
And then look at the RN gallery for Windows examples specifically: https://github.com/microsoft/react-native-gallery
